e2e/test/helpers/RetryOperationHelper.cs

```diff
@@ -51,5 +52,43 @@ public static async Task RetryOperationsAsync(Func<Task> asyncOperation, IRetryP
             }
             while (shouldRetry);
         }
+
+        /// <summary>
+        /// Rety an async operation based on the retry strategy supplied.
+        /// </summary>
+        /// <param name="asyncOperation">The async operation to be retried.</param>
+        /// <param name="retryPolicy">The retry policy to be applied.</param>
+        /// <param name="retryableStatusCodes">The errors to be retried on.</param>
+        /// <param name="logger">The <see cref="MsTestLogger"/> instance to be used.</param>
+        /// <returns></returns>
```

Comment:
Fill in doc comment or remove if it is not interesting (as is the case with `Task`).